versions/1/data_stream/spec.yml

```diff
@@ -1,10 +1,10 @@
 spec:
-- description: Folder containing a single dataset definition
+- description: Folder containing a single data stream definition
```

Comment:
Stumbled over this as it is not fully current. This a template for all data streams created for a specific `type + dataset` combination. All `logs-nginx.access-*` will use this template, independent of the namespace.

In my head, these are data stream templates.

Reply:
I'm afraid I'm lost here. @ruflin, could you please elaborate more about this? data streams, templates, namespaces

Reply:
Yes, I'm lost too. I think it might be useful to add a `definitions.md` or just a `README.md` file in the `versions/1` folder defining all these terms and concepts and how they relate to each other. The spec (so far) describes the syntactic relationships but I also have to admit that I don't fully understand all the semantic relationships between these concepts.

@ruflin can you please make a new PR to add such a file? I think you are the only(?) one who understands all these terms (old and new), the corresponding concepts, and the relationships between them.